chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -357,7 +358,21 @@
     --is-public True \
     --page-size 1 \
     2> /dev/null | tail -n 2 | head -n 1 | awk '{ print $2 }')
-[ -n "$DOCKER_IMAGE_ID" ] && echo "$DOCKER_IMAGE_ID" > #{docker_image_id_file}
+if [ -n "$DOCKER_IMAGE_ID" ] ; then
+  echo "$DOCKER_IMAGE_ID" > #{docker_image_id_file}
+else
+  for node in #{docker_nodes}; do
+      ssh $node docker pull cirros
+  done
+  DOCKER_IMAGE_ID=$(glance #{insecure} image-create --name #{image_name} \
+      --container-format docker \
+      --disk-format raw --property hypervisor_type=docker \
+      --is-public True  \
+      --copy-from http://clouddata.cloud.suse.de/images/docker/cirros.tar \
```

Comment:
We can't put this URL in the barclamp; this needs to work for external people also.

We could possibly have the URL in an attribute, and then set the attribute in mkcloud, though.